nntrainer/layers/lstm.cpp

```diff
@@ -153,7 +170,22 @@ void LSTMLayer::forwarding(bool training) {
   Tensor &bias_h =
     weightAt(static_cast<int>(LSTMParams::bias_h)).getVariableRef();
 
-  Tensor &hidden_ = net_hidden[0]->getVariableRef();
+  mem_cell->getVariableRef().setZero();
+  hidden->getVariableRef().setZero();
+  fgio->getVariableRef().setZero();
+
+  if (training) {
+    mem_cell->getGradientRef().setZero();
```

Comment:
IMO this should be moved to `calcGradient()`.

Reply:
done.